tests/dataset.py

```diff
@@ -51,6 +51,7 @@ def to_vector(self):
 bidon = Bidon()
 bidon_hlist = Dataset([bidon] * 10)
 bidon_hlist.plot_data()
+print(bidon_hlist)
```

Comment:
Has this print been let there on purpose or has it been forgotten ?

Reply:
It is here on purpose but maybe you would like me to write an assertion ?

Reply:
Fully asserting string equalities can be a bit overkill. Why not asserting the good length of the string ?

If the print is for checking purpose, chances are it will never be manually checked in CI process, so we should assert something, indeed

Reply:
I made a test